base/shared/src/main/scala/scalaz/data/package.scala

```diff
@@ -1,7 +1,7 @@
 package scalaz
 
 package object data {
-  val Void: VoidModule with VoidSyntax = VoidImpl
+  val Void: VoidModule = VoidImpl
 
   val Forall: ForallModule with ForallSyntax = ForallImpl
```

Comment:
There is also `Forall` and `Forall2`, which use the same encoding.

Reply:
@alexknvl,
Let me know if the latest commit is what you had in mind.

Reply:
We don't need ` with ForallSyntax` here anymore.